modules/metering-reports.adoc

```diff
@@ -48,14 +50,15 @@ spec:
   reportingEnd: "2019-07-31T00:00:00Z"
 ----
 
+[id="metering-query_{context}"]
```

Comment:
Consider adding some transitional language here - I'm not sure how we got from sample reports to "query." Is this the start of some command or parameter examples? Do you need another H2 and to make this heading and the following headings H3s? (If this is reference content, consider moving it to its own module.)